### PR TITLE
add bio requested status to report page

### PR DESCRIPTION
**Pull request by juneluo**
### Description
Add a “Bio Posted/Requested” selection to the Reports page
This is so anyone looking at the reports page knows if a person’s bio announcement is in the process of being created or already created.  
Changing this should auto-save (show saving notification like saving an intangible time log) but not refresh the page 
It should be editable by Owner/Admin class only

### Mainly changes explained:
Add a switch toggle which can only view and edited by Admin/Owner in the people report page. Add a line of text showing the current bio status that can be viewed by anyone.
PS: many lines of changes are just brought by formatting.

### How to test:
**Please test with the [back end PR](https://github.com/OneCommunityGlobal/HGNRest/pull/346).**
Login in as a Owner/Admin: dashboard-->report-->people-->click any user

Login in as a other role: dashboard-->report-->people-->click any user

### CODE REVIEW

**Review by abdel-lall — APPROVED** [review]
Works as expected. Nice job!

PR807

**Review by GKD890 — APPROVED** [review]
Hi, it works as properly. I tried as `admin` and `manager` . code also looks good to me.

Frontend_PR#807

**Review by wang9hu — APPROVED** [review]
Looks great! The bio status changes as expected!

Volunteer user:
Screenshot 2023-05-06 204347

Admin user:
Screenshot 2023-05-06 205337

Volunteer user (after change)
Screenshot 2023-05-06 205530

**Comment by one-community** [comment]
Thank you everyone, moving this to final review!

**Comment by one-community** [comment]
Excellent work on this. DMd you with followup requests for this.

**Review by EdwinLauM — APPROVED** [review]
Works as intended. Tested toggling with admin's account and changes were reflected in volunteer's account! Good job!

Before toggling volunteer's account
Testing PR #807   346 before


Toggling with admin's account
Testing PR #807   346 change

After toggling volunteer's account
Testing PR #807   346 after

**Review by alexander-huerta — APPROVED** [review]
Nice work! 

I tested as an Admin and Manager. 

1. As an admin: Reports->People-> clicked into a Volunteer and toggled the status from "requested "to "posted". 
2. It is automatically saved, without refreshing the page
3. As a manager: Reports->People-> the same Volunteer 
4. The status showed correctly as “Bio Posted”

Before Toggle (admin on the left half of the screenshot)
807-346_requested


After Toggle / changes saved  (admin on the left half of the screenshot)

807-346_posted

**Review by AuroraLovesWhales — APPROVED** [review]
I have tested this PR and it is working as expected. Great job!
Log in as an admin and click on another user:
Before toggling:
Pasted Graphic 15

After toggling:
Pasted Graphic 16

Log in as a volunteer and click on another user:
Before toggling:
Pasted Graphic 14

After toggling, the new status is reflected as expected:
Pasted Graphic 17

File: src/components/Reports/PeopleReport/PeopleReport.jsx [hunk 1]
```diff
@@ -460,24 +440,53 @@ class PeopleReport extends Component {
...
             <p>Start Date</p>
           </div>
           <div>
-            <h4>{userProfile.endDate ? userProfile.endDate.toLocaleString().split('T')[0] : 'N/A'}</h4>
+            <h4>
+              {userProfile.endDate ? userProfile.endDate.toLocaleString().split('T')[0] : 'N/A'}
+            </h4>
             <p>End Date</p>
           </div>
+          <div>
```
Comment (EvianTan) [inline src/components/Reports/PeopleReport/PeopleReport.jsx:454]: Functional changes from here.

**Review by EvianTan — APPROVED** [review]

**Pull request merged**